platform/src/termination.rs

```diff
@@ -0,0 +1,25 @@
+//! Definition of the Termination trait which must be implemented
+//! by any type returned by a function declared as an application
+//! entry point via `set_main!()`;
```

Comment:
This wording is kinda awkward. How about something like:

Definition of the `Termination` trait. The `main()` function (set using `set_main!()`) must return a type that implements `Termination`.